pkg/search/postgres/common.go

```diff
@@ -89,7 +92,10 @@ type query struct {
 	Pagination parsedPaginationQuery
 	InnerJoins []innerJoin
 
-	GroupBys []groupByEntry
+	// This indicates if a primary key is present in the group by clause. Unless GROUP BY clause is explicitly provided,
+	// we order the results by the primary key of the schema.
```

Comment:
It might be worthwhile to explore sql proto tags which indicate default group by field.  This will help in cases where the ID is a uuid but the unique name is almost always what we want to group by (category name, policy name, cluster name)

Reply:
I don't think we should add query information to protos. Protos carry the schema definition whereas select query is a custom user-defined operation. I would like to keep the contract as transparent as possible. Users must specify a complete query for the response they expect.